src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawController.java

```diff
@@ -6,13 +6,14 @@
 import edu.kis.powp.jobs2d.Job2dDriver;
 
 /**
- * driver adapter to drawer with several bugs.
+ * Driver adapter to drawer.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawController implements Job2dDriver {
```

Comment:
The name does not indicate the purpose of the adapter or what was actually adapted.